CMakeLists.txt

```diff
@@ -52,14 +52,16 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 set(configure_warnings)
-if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
-  include(CheckPIESupported)
-  check_pie_supported(OUTPUT_VARIABLE check_pie_output LANGUAGES CXX)
-  if(NOT CMAKE_CXX_LINK_PIE_SUPPORTED)
-    list(APPEND configure_warnings "PIE link options are not supported for executable targets: ${check_pie_output}.")
+if(NOT CMAKE_CROSSCOMPILING)
```

Comment:
I'm not sure what cross compiling has to do with PIE? What if I'm building for Linux from Windows? Or from Linux for *BSD?

See here for how this logic should look imo: https://github.com/hebasto/bitcoin/pull/10#issuecomment-1452669584

Reply:
> See here for how this logic should look imo: [#10 (comment)](https://github.com/hebasto/bitcoin/pull/10#issuecomment-1452669584)

Done in https://github.com/hebasto/bitcoin/pull/10#issuecomment-1466596369.